### ISSUE TITLE
import is duplicated when using `import` and `import type`

### ISSUE THREAD

**Issue by Gozala** [gajus/eslint-plugin-flowtype#72]
I am using this plugin in conjunction with [standard](http://standardjs.com) and run into issue with code like this:

``` js
import type {Action} from "./module"
import {init, view, update} from "./module"
```

I can not include Action in the same import as flow would complain that Action type is imported as value but if I don't then standard complains about duplicate errors.

**Comment by danharper** [comment]
This isn't an issue with this plugin, it's because ESLint's normal `no-duplicate-imports` doesn't support Flow type imports.

Use [`eslint-plugin-import`](https://github.com/benmosher/eslint-plugin-import) with their [`import/no-duplicates`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md) rule instead.

I don't use Standard, so not sure how much you can configure that.

**Comment by Gozala** [comment]
> This isn't an issue with this plugin, it's because ESLint's normal no-duplicate-imports doesn't support Flow type imports.

Depends how you look at it I guess. Without this plugin ESLint fails to parse files with flow annotations all together. With this plugin it can parse flow type annotations, although it's unable to distinguish type imports from regular imports. I would expect that plugin that adds support for flow syntax extensions should also make ESLint distinguish (or maybe ignore) type imports.

> Use eslint-plugin-import with their import/no-duplicates rule instead.

I guess adding something along the lines to what this patch did to eslint-plugin-imports https://github.com/benmosher/eslint-plugin-import/pull/334/files does would make most sense. But then again I have very little understanding how all this plugin hooks run or what they can do.

> I don't use Standard, so not sure how much you can configure that.

You can not configure standard at all.

**Comment by danharper** [comment]
> Without this plugin ESLint fails to parse files with flow annotations all together.

This plugin doesn't parse Flow at all, that's all with `babel-eslint`.

**Comment by danharper** [comment]
See here for some more info: https://github.com/babel/babel-eslint/issues/293

**Comment by feross** [comment]
Can this package rewrite type imports into another import line so this will work with eslint out of the box?

For example, this:

```js
import type {Action} from "./module"
import {init, view, update} from "./module"
```

Would turn into this:

```js
import {Action, init, view, update} from "./module"
```

The only level of customization that `standard` supports is specifying a plugin. No rules can be enabled/disabled. So it would be nice if just the presence of this plugin was enough to get flowtype working with `standard`.

**Comment by gajus** [comment]
Solution provided by https://github.com/gajus/eslint-plugin-flowtype/issues/72#issuecomment-240851652
